### DISCUSSION TITLE
Claiming IOTA lottery funds issue

### DISCUSSION THREAD

**Discussion by drgunlove** [iotaledger/firefly#8089] [Help & Support]
Hi friends, I have an issue that has left me slightly confused. I logged into my FF wallet today and noticed that I had a notification of an IOTA lottery win. which showed in my activity feed. I clicked on this and followed the prompts and then set up a Firefly shimmer wallet. There was no mention of being able to claim the lottery and after consolidating my iota firefly wallet to the new shimmer wallet, I have less funds in my IOTA firefly wallet and nothing in my Shimmer wallet. Have I done something wrong? Where have my Iota firefly wallet funds go? Is this lottery notification a scam? I am totally confused as I just followed the prompts from IOTA of what to do. Any help greatly appreciated.

**Comment by begonaalvarezd** [comment]
Scam Alert!

Currently, scammers are sending NFTs to IOTA addresses that aim to trick users into visiting an "IOTA Lottery" website where it is promised they could claim their "Win".

There is no IOTA Lottery, and visiting this website can lead to losing all Funds in your IOTA Wallet.

It is not possible to block receiving these NFT so either ignore them or "burn" them by following these steps:
1) In your wallet go to: Collectibles
2) Choose the scammer NFT
3) Click on the three dots on the top right
4) Burn NFT

NEVER enter your seed, 24 words Phrase or backup file in ANY website or online application.

Stay safe.

If you fell victim to this scam please report it to your local authorities.

![image](https://github.com/iotaledger/firefly/assets/3624944/ff6e990d-c24d-4a8a-95e6-f3898b345f17)

**Comment by begonaalvarezd** [comment]
Im am very sorry you are a victim of this scam, we cant prevent people from gifting NFTs to others and in this case you were gifted an NFT which unfortunatelly has a website in the description which is a scam website. Self custody wallets in crypto are about users being the only ones in control of their keys, so you should NEVER share them nor enter them in random websites. 

If you have vesting airdrop rewards you can try being faster than the scammer and when the funds unlock send them to another wallet that is not compromised.

Stay safe
